transformations/synonym_substitution/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "synonym_substitution",
+  "test_cases": [
+    {
+      "class": "SynonymSubstitution",
+      "inputs": {
+        "sentence": "Andrew finally returned the French book to Chris that I bought last week"
+      },
+      "outputs": {
```

Comment:
Please take a master pull, outputs should list of dict.

Reply:
Updated.